apps/examples/testcase/le_tc/kernel/tc_pthread.c

```diff
@@ -1554,6 +1607,9 @@ int pthread_main(void)
 	tc_pthread_pthread_set_get_schedparam();
 #if CONFIG_NPTHREAD_KEYS > 0
 	tc_pthread_pthread_key_create_delete_set_getspecific();
+#endif
+#ifndef CONFIG_PTHREAD_MUTEX_UNSAFE
+	tc_pthread_pthread_mutex_consistent();
```

Comment:
I think we'd better move this tc after `tc_pthread_pthread_mutex_lock_unlock_trylock()`.

Reply:
yeah making changes